examples/tokio.rs

```diff
@@ -8,24 +8,25 @@
 //!
 //! You really do want better error handling than all these unwraps.
 
-extern crate futures;
-extern crate tokio_core;
-extern crate tun_tap;
-
+use bytes::Bytes;
+use futures_util::stream::StreamExt;
+use futures_util::SinkExt;
 use std::process::Command;
 use std::time::Duration;
+use tokio::time::sleep;
+use tokio_util::codec::{BytesCodec, Decoder};
 
-use futures::{Future, Stream};
-use tokio_core::reactor::{Core, Interval};
+use tun_tap::r#async::Async;
```

Comment:
Style nitpick. It looks a bit strange to intermix blocks of "std" imports with blocks of "foreign". My own personal preference is to have 3 blocks (std, foreign, internal), but I'm flexible in the order of these… and I try to keep each separate block sorted.

Reply:
Also, as `async` is now a keyword, and you seem to have modified the API anyway, it would be a good time to think of a better name.